api/src/websocket/services/socket-event-dispatcher.service.ts

```
@@ -74,25 +75,31 @@ export class SocketEventDispatcherService implements OnModuleInit {
-        req.session.reload((_err) => {
-          if (_err) reject();
-          resolve();
+      await new Promise<Error | void>(async (resolve, reject) => {
+        req.session.reload((error) => {
+          if (error) {
+            reject(new UnauthorizedException());
+          } else {
+            resolve();
+          }
         });
       });
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)